### DISCUSSION TITLE
Date Alignment Time Series Chart

### DISCUSSION THREAD

**Discussion by daniel10012** [apache/superset#18466] [Ideas]
![image](https://user-images.githubusercontent.com/10360991/112086280-4d9b2080-8b62-11eb-9596-fee512442099.png)

![image](https://user-images.githubusercontent.com/10360991/112086332-61df1d80-8b62-11eb-9db0-a1545bdf4895.png)

@junlincc

**Comment by junlincc** [comment]
Echarts actually have a very intuitive out-of-the-box solution to display x ticks. we removed it in favor of the customized date format. Opening a PR to bring it back and set it dynamic by default. but it will still have overlaps when user changes the date format....a more throughout change need to come from the Echarts team, which we have requested to. 

@daniel10012 @eugeniamz
